### DISCUSSION TITLE
Jobs not running in production

### DISCUSSION THREAD

**Discussion by xts-bit** [taskforcesh/bullmq#2119] [Q&A]
I'm trying to run a job using Bullmq the jobs are scheduled to run on the next day, But the issue is the jobs are not running, and in my production server, the sever restarted because I pushed some new code. 

Can anyone please help me with this?

```
import { Queue, Worker } from 'bullmq';
import { Redis } from 'ioredis';

const redisClient = new Redis({
    host: 'myhost',
    port: 3806,
    password: 'mypass',
    maxRetriesPerRequest: null,
    enableReadyCheck: false,
    retryStrategy: function (times) {
        return Math.max(Math.min(Math.exp(times), 20000), 1000);
    }
});

const queue = new Queue('myQueue', {
    connection: redisClient,
    enableOfflineQueue: false,
});

const worker = new Worker('myQueue', async (job) => {
   await hitApi()
}, {
    connection: redisClient,
});

async function hitApi() {
    console.log("Working")
}
queue.add("myJob", { foo: "bar" }, { delay: delay });

worker.on('completed', (job) => {
    console.log(`Job completed`);
});

worker.on('failed', (job, err) => {
    console.error(`Job failed`, err);
})
```

**Comment by roggervalf** [comment]
hi @xts-bit, in your example I cannot see the delay value and if you are only restarting your server, it shouldn't affect your redis db instance unless your are also flushing it. Having a ui to see your jobs records could help on seeing if your jobs are saved or just use redis-cli

**Reply by xts-bit** [reply]
@roggervalf Rogger, Please tell what UI can i show from where?

**Reply by xts-bit** [reply]
@roggervalf The delay value is the next day which i get from a function that generates a random time of the next day. for example 1691380746.81

**Reply by xts-bit** [reply]
@roggervalf What more information can I provide to you for helping me fix this issue?

**Reply by xts-bit** [reply]
Is there any possible way to fix these problems?

**Reply by manast** [reply]
@xts-bit if you have an issue to report please follow the bug report with clear instructions on how to reproduce said issue.
